### PR TITLE
Trunk  misc

### PR DESCRIPTION
**Pull request by ghost**
Trivial, unrelated, low-priority (but nevertheless useful) commits.
(I am making a pull-request only to avoid losing them by waiting.)

### CODE REVIEW

File: kernel/pre_env.mli [hunk 1]
```diff
@@ -47,7 +47,7 @@ type named_context_val = private {
 
 type env = {
     env_globals       : globals;
-    env_named_context : named_context_val;
+    env_named_context : named_context_val;  (** in the interactive theorem proving mode, we see these above ============================ *)
```
Comment (ppedrot) [inline kernel/pre_env.mli]: I doubt this is the proper place to document this, `pre_env` is an internal structure of `Environ.env`. Worse, it is tied to the way we display things in coqtop, which is totally unconnected to the kernel. You should probably remove that comment and describe the structure in a separate document.

**Review by ppedrot — CHANGES_REQUESTED** [review]

File: kernel/pre_env.mli (hunk 1 above)
Comment (matejkosik) [inline kernel/pre_env.mli]: Commit removed.

What is the right place to explain the meaning of `Pre_env.env`?

**Comment by matejkosik** [comment]
I've removed the problematic commit. What needs to be done next?

**Review by ppedrot — APPROVED** [review]

**Comment by matejkosik** [comment]
Related Travis tests [are running](https://travis-ci.org/matejkosik/coq/builds/237825876).

**Pull request merged**